dm/config/task.go

```diff
@@ -66,6 +68,11 @@ var (
 	defaultBatch                   = 100
 	defaultQueueSize               = 1024 // do not give too large default value to avoid OOM
 	defaultCheckpointFlushInterval = 30   // in seconds
+
+	// TargetDBConfig
+	defaultSessionCfg = map[string]string{
+		tidbTxnMode: tidbTxnOptimistic,
```

Comment:
Does a lower version of TiDB without optimistic works well (or without interrupt)?